src/lib/components/primitives/select/select.stories.js

```diff
@@ -54,3 +56,28 @@ export const LabelLast = {
     defaultOption: null,
   },
 };
+
+const Template = (args) => ({
+  Component: Select,
+  props: args,
+});
+
+export const Interaction = Template.bind({})
+
+Interaction.args = { ...LabelFirst.args };
+Interaction.play = async ({ canvasElement }) => {
+
+  const canvas = within(canvasElement);
+  const label = canvas.getByLabelText('Favorite Color');
+
+
+  await userEvent.tab();
+  expect(label).toHaveFocus()
+
+  await userEvent.selectOptions(label, ['0'])
+  expect(canvas.getByText('Red')).toBeTruthy()
+  await userEvent.selectOptions(label, ['1'])
+  expect(canvas.getByText('Green')).toBeTruthy()
+  await userEvent.selectOptions(label, ['2'])
+  expect(canvas.getByText('Blue')).toBeTruthy()
```

Comment:
I'm unsure about the value being tested here. I do feel this is testing native browser behavior, but I also think the assertions will never fail. Selecting an option doesn't remove the other options text from the DOM, I could be wrong, but regardless, I think this may distract us from a more important feature of the component to test.

This component has custom logic that we wrote that is much more likely to fail than a native browser element. For example:

1) Let's make sure a `defaultValue` passed into the component is indeed selected appropriately when rendered

2) Make sure the auto-keyboard focus works as expected. See here:
https://github.com/cerebrl/forgerock-web-login-framework/blob/main/src/lib/components/primitives/select/select.svelte#L28

    The above logic means if this input is invalid, it must apply keyboard focus to itself as it's the first of a list of invalid inputs. This would be good to test.

3) We need to ensure our event handler and `onChange` function passed in all are called as expected. See here: https://github.com/cerebrl/forgerock-web-login-framework/blob/main/src/lib/components/primitives/select/select.svelte#L34

4) We also have the small bit of logic that handles if the `null` option has the same text as the label, it doesn't display the label as it's redundant.

In my opinion, the above have a much higher ROI as we are more likely to break the custom logic we have than a browser breaking how a `select` element works.